services/updater/updater.go

```diff
@@ -215,7 +229,6 @@ func (updater *UpdaterService) compareScannerNodeVersion(previousRef string) (ne
 
 func (updater *UpdaterService) checkNewerReleaseAndWait(previousRef string, delay time.Duration) (foundNew bool) {
 	detectedCh := make(chan struct{})
-	defer close(detectedCh)
```

Comment:
This was causing a closed channel send panic. Removing it as the channel is deallocated anyways.